config/management/operational/src/command.rs

```diff
@@ -103,6 +115,20 @@ impl Command {
         }
     }
 
+    pub fn add_validator(self) -> Result<TransactionContext, Error> {
+        match self {
+            Command::AddValidator(cmd) => cmd.execute(),
+            _ => Err(self.unexpected_command(CommandName::AddValidator)),
+        }
+    }
+
+    pub fn remove_validator(self) -> Result<TransactionContext, Error> {
+        match self {
+            Command::RemoveValidator(cmd) => cmd.execute(),
+            _ => Err(self.unexpected_command(CommandName::RemoveValidator)),
+        }
+    }
```

Comment:
@davidiw Any idea on how we could shrink this boilerplate stuff that we've put for every command?

Reply:
yeah I hate this

Reply:
A macro, where you pass in the right side of Command/CommandName and the return type. We'd need to convert the command to underscore case and then you could have this in 1 line.